code/modules/surgery/organs/subtypes/standard.dm

```diff
@@ -136,9 +136,9 @@
 		if(owner.gloves)
 			owner.unEquip(owner.gloves)
 		if(owner.l_hand)
-			owner.unEquip(owner.l_hand,1)
+			owner.unEquip(owner.l_hand)
 		if(owner.r_hand)
-			owner.unEquip(owner.r_hand,1)
+			owner.unEquip(owner.r_hand)
```

Comment:
This stuff is to prevent the guard SNPCs from dropping their guns under any circumstances. What was the reason these were force-unequip?

Reply:
Presumably because if you cut off a traitor's hands when they're wielding a chainsaw, it should disobey nodrop